engine/glfw/lib/android/android_joystick.c

```diff
@@ -480,3 +480,9 @@ int _glfwPlatformGetJoystickDeviceId( int joy, char** device_id )
         return GL_TRUE;
     }
 }
+
+int _glfwPlatformGetJoystickGenericDeviceId( int joy, char** generic_device_id )
```

Comment:
Do nothing. Return the device id.

Future update: I think we can potentially do something similar to the html5 Standard Gamepad on Android.